<!DOCTYPE html>
<html lang="en">
<head>
  <!-- hexo-inject:begin --><!-- hexo-inject:end --><meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 5.0.0">
  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32-next.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16-next.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#222">

<link rel="stylesheet" href="/css/main.css">


<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">

<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"panyuwen.github.io","root":"/","scheme":"Muse","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"always","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":true,"show_result":false,"style":"mac"},"back2top":{"enable":true,"sidebar":false,"scrollpercent":true},"bookmark":{"enable":true,"color":"#222","save":"manual"},"fancybox":false,"mediumzoom":false,"lazyload":false,"pangu":false,"comments":{"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":true,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}},"path":"search.xml"};
  </script>

  <meta name="description" content="Effective population size ($N_e$)PrefaceThe effective population size ($N_e$) is the number of individuals in a theoretically ideal population having the same magnitude of some specified quantity of i">
<meta property="og:type" content="article">
<meta property="og:title" content="7. effective population size">
<meta property="og:url" content="https://panyuwen.github.io/2020/08/21/7-effective-population-size/index.html">
<meta property="og:site_name" content="KNOCKIN on HEAVEN&#39;s DOOR">
<meta property="og:description" content="Effective population size ($N_e$)PrefaceThe effective population size ($N_e$) is the number of individuals in a theoretically ideal population having the same magnitude of some specified quantity of i">
<meta property="og:locale" content="en_US">
<meta property="og:image" content="https://github.com/panyuwen/figs/blob/master/fixation_index.png?raw=true">
<meta property="article:published_time" content="2020-08-21T15:53:38.000Z">
<meta property="article:modified_time" content="2020-08-23T15:15:36.235Z">
<meta property="article:author" content="KNOCK KNOCK KNOCKIN">
<meta property="article:tag" content="population genetics">
<meta property="article:tag" content="diversity">
<meta name="twitter:card" content="summary">
<meta name="twitter:image" content="https://github.com/panyuwen/figs/blob/master/fixation_index.png?raw=true">

<link rel="canonical" href="https://panyuwen.github.io/2020/08/21/7-effective-population-size/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : false,
    isPost : true,
    lang   : 'en'
  };
</script>

  <title>7. effective population size | KNOCKIN on HEAVEN's DOOR</title>
  






  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript><!-- hexo-inject:begin --><!-- hexo-inject:end -->

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <!-- hexo-inject:begin --><!-- hexo-inject:end --><div class="container use-motion">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="Toggle navigation bar">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">KNOCKIN on HEAVEN's DOOR</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
      <p class="site-subtitle" itemprop="description">to record something</p>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
        <i class="fa fa-search fa-fw fa-lg"></i>
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>Home</a>

  </li>
        <li class="menu-item menu-item-archives">

    <a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>Archives</a>

  </li>
      <li class="menu-item menu-item-search">
        <a role="button" class="popup-trigger"><i class="fa fa-search fa-fw"></i>Search
        </a>
      </li>
  </ul>
</nav>



  <div class="search-pop-overlay">
    <div class="popup search-popup">
        <div class="search-header">
  <span class="search-icon">
    <i class="fa fa-search"></i>
  </span>
  <div class="search-input-container">
    <input autocomplete="off" autocapitalize="off"
           placeholder="Searching..." spellcheck="false"
           type="search" class="search-input">
  </div>
  <span class="popup-btn-close">
    <i class="fa fa-times-circle"></i>
  </span>
</div>
<div id="search-result">
  <div id="no-result">
    <i class="fa fa-spinner fa-pulse fa-5x fa-fw"></i>
  </div>
</div>

    </div>
  </div>

</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>
  <a role="button" class="book-mark-link book-mark-link-fixed"></a>

  <a href="https://github.com/panyuwen" class="github-corner" title="Follow me" aria-label="Follow me" rel="noopener" target="_blank"><svg width="80" height="80" viewBox="0 0 250 250" aria-hidden="true"><path d="M0,0 L115,115 L130,115 L142,142 L250,250 L250,0 Z"></path><path d="M128.3,109.0 C113.8,99.7 119.0,89.6 119.0,89.6 C122.0,82.7 120.5,78.6 120.5,78.6 C119.2,72.0 123.4,76.3 123.4,76.3 C127.3,80.9 125.5,87.3 125.5,87.3 C122.9,97.6 130.6,101.9 134.4,103.2" fill="currentColor" style="transform-origin: 130px 106px;" class="octo-arm"></path><path d="M115.0,115.0 C114.9,115.1 118.7,116.5 119.8,115.4 L133.7,101.6 C136.9,99.2 139.9,98.4 142.2,98.6 C133.8,88.0 127.5,74.4 143.8,58.0 C148.5,53.4 154.0,51.2 159.7,51.0 C160.3,49.4 163.2,43.6 171.4,40.1 C171.4,40.1 176.1,42.5 178.8,56.2 C183.1,58.6 187.2,61.8 190.9,65.4 C194.5,69.0 197.7,73.2 200.1,77.6 C213.8,80.2 216.3,84.9 216.3,84.9 C212.7,93.1 206.9,96.0 205.4,96.6 C205.1,102.4 203.0,107.8 198.3,112.5 C181.9,128.9 168.3,122.5 157.7,114.1 C157.9,116.9 156.7,120.9 152.7,124.9 L141.0,136.5 C139.8,137.7 141.6,141.9 141.8,141.8 Z" fill="currentColor" class="octo-body"></path></svg></a>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          

          <div class="content post posts-expand">
            

    
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="en">
    <link itemprop="mainEntityOfPage" href="https://panyuwen.github.io/2020/08/21/7-effective-population-size/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/IMG_F7DE10EB3810-1.jpeg">
      <meta itemprop="name" content="KNOCK KNOCK KNOCKIN">
      <meta itemprop="description" content="life, study, and work">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="KNOCKIN on HEAVEN's DOOR">
    </span>
      <header class="post-header">
        <h1 class="post-title" itemprop="name headline">
          7. effective population size
        </h1>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">Posted on</span>

              <time title="Created: 2020-08-21 23:53:38" itemprop="dateCreated datePublished" datetime="2020-08-21T23:53:38+08:00">2020-08-21</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">Edited on</span>
                <time title="Modified: 2020-08-23 23:15:36" itemprop="dateModified" datetime="2020-08-23T23:15:36+08:00">2020-08-23</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">In</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/statistics/" itemprop="url" rel="index"><span itemprop="name">statistics</span></a>
                </span>
            </span>

          
            <span class="post-meta-item" title="Views" id="busuanzi_container_page_pv" style="display: none;">
              <span class="post-meta-item-icon">
                <i class="fa fa-eye"></i>
              </span>
              <span class="post-meta-item-text">Views: </span>
              <span id="busuanzi_value_page_pv"></span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
        <h1 id="Effective-population-size-N-e"><a href="#Effective-population-size-N-e" class="headerlink" title="Effective population size ($N_e$)"></a>Effective population size ($N_e$)</h1><h2 id="Preface"><a href="#Preface" class="headerlink" title="Preface"></a>Preface</h2><p>The effective population size ($N_e$) is the number of individuals in a theoretically ideal population having the same magnitude of some specified quantity of interest (e.g., genetic drift) as the actual population under the <code>Wright-Fisher Model</code>. </p>
<a id="more"></a>

<p>There are different kinds of effective population size based on the way how we measure “magnitude”, including (1) <code>inbreeding effective population size</code>: the change in probability of identity by descent ($F$), (2) <code>variance effective population size</code>: the change in variance in allele frequency, and so on. </p>
<blockquote>
<p><a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Effective_population_size">https://en.wikipedia.org/wiki/Effective_population_size</a></p>
<p>Principle of population genetics (4th) </p>
</blockquote>
<h2 id="Fixation-index"><a href="#Fixation-index" class="headerlink" title="Fixation index"></a>Fixation index</h2><p>Two alleles are identical by descent (IBD) (due to finite population size) if they are replicas of a gene present in sample previous generation (choose come arbitrary time in the past). The concept of IBD allows us to distinguish two types of homozygous genotypes (e.g., IBD AA and not IBD AA). In some cases, alleles may be indistinguishable (their status in regard to identity by descent is unknown), such alleles are identical by state (IBS).   </p>
<p>The probability that the alleles in an individual are <code>IBD</code> is often denoted $F$ (Fixation index). The probability of allele IBD in generation $t$ is represented as $F_t$.  </p>
<p>When the population subdivision first take place, and all subpopulations have the same allele frequencies ($t=0, F_t=0$). As time goes on, each subpopulation undergoes random drift, the genotype frequencies in each subpopulation will statisfy the Hardy-Weinberg principle. However, the allele frequencies among the subpopulations will change because of random genetic drift, and moreover the value of $F_t$ will gradually increase as more and more alleles within any subpopulations become IBD owing to common ancestry.  </p>
<p><img src="https://github.com/panyuwen/figs/blob/master/fixation_index.png?raw=true"></p>
<p>There are $2N$ alleles in a breeding population of generation $t-1$. In sampling alleles for generation t, the first allele chosen may be any of those present in generation $t-1$ with equal probability. Then the probability that the second allele chosen in of the same type as the first is $\frac{1}{2N}$ (in which case $F=1$), which is the frequency of each allelic type in the pool of gamates. And the propability that the second chosen allele is of a different type is accordingly $1-\frac{1}{2N}$ (in which case $F=F_{t-1}$). We get </p>
<p>$F_t=\frac{1}{2N}+(1-\frac{1}{2N})F_{t-1}$, </p>
<p>which can be regarded as the probability of either being autologous in the present generation <code>OR</code> the probability that an individual comes from that fraction of the population that is already autozygous due to inbreeding in the previous generation.   </p>
<p>Multiplying both sides by $-1$ and then adding $1$ to each side leads to </p>
<p>$1-F_t=1-\frac{1}{2N}-(1-\frac{1}{2N})F_{t-1}=(1-\frac{1}{2N})(1-F_{t-1})$</p>
<p>$1-F_t=(1-\frac{1}{2N})^t(1-F_0) = e^{\frac{-t}{2N}} * (1-F_0)$</p>
<p>$1-F_t=(1-\frac{1}{2N})^t = e^{\frac{-t}{2N}}$, when $F_0=0$ . </p>
<p>Since $F_t$ is the probability that 2 alleles in an individual in generation $t$ are IBD, and assuming $p_0$ is the overall allele frequency if $A$ averaged across all subpopulations, the propability that a randomly chosen individual will be genotypically $AA$ is </p>
<p>$p_0^2(1-F_t)$ [$P(A)*P(A_{non-IBD})$]$+p_0F_t$ [$P(A)*P(IBD)$] $ = p_0^2(1-F_t)+p_0F_t$ . </p>
<p>Similarly, the average genotype frequencies among subpopulations at any time $t$ have the expected values: </p>
<p>$AA: p_0^2(1-F_t)+p_0F_t = p_0^2+p_0q_0F_t$</p>
<p>$Aa: 2p_0q_0(1-F_t) = 2p_0q_0-2p_0q_0F_t$</p>
<p>$aa: q_0^2(1-F_t)+q_0F_t = q_0^2+p_0q_0F_t$ .</p>
<p>At any time, the average frequency of heterozygous genotypes among the subpopulations, $H_t$, relative to what it would be without population subdivision, $H_0$, decreases linearly in $F_t$. Hence we have $H_t=(1-F_t)H_0$, where $H_0=2p_0q_0$. We obtain $H_t=(1-\frac{1}{2N})^tH_0 \approx H_0e^{\frac{-t}{2N}}$. That is, in each successive generation, the heterozygosity ($H$) declines by $\frac{1}{2N}$. And the smaller the population size ($N$), the faster the decline in heterozygosity.   </p>
<p>Finally, $F_t=1$ and there are no heterozygotes left ($H=0$), and all subpopulations are fixed. The allele that is already more frequent will have a higher probability of being fixed. And the probability of a neutral allele being fixed equals to its frequency. </p>
<blockquote>
<p>P.S.</p>
<p>Genetic drift $D \approx 1-e^{\frac{-t}{2N}} \approx 2F_{ST}$  </p>
<p>$N = \frac{-t}{2 \ * \ ln(1-2F_{ST})}$  (would be useful)  </p>
</blockquote>
<h2 id="The-mutation-drift-balance"><a href="#The-mutation-drift-balance" class="headerlink" title="The mutation-drift balance"></a>The mutation-drift balance</h2><p>Each generation, $2N\mu$ new mutations added, which thus arise at an initial frequency of $\frac{1}{2N}$. For each of these, $\frac{1}{2N}$ probability of eventually going to fixation (by chance along, if it is neutral to selection), since the probability that an allele will be fixed is equal to its frequency. Thus, the total number of mutations that will be fixed per generation is $2N\mu* \frac{1}{2N}=\mu$ (the probability of getting a new mutant in the population and then the probability that this mutant will be fixed), while the rate (time interval) at which new neutral mutants are fixed is $\frac{1}{\mu}$, independent of population size $N$. However, the time it takes a particular mutant to achieve fixation from the time it arises is dependent on populaiton size ($4N_e$ generations).   </p>
<p>In the absence of mutation, the probability of being autozygous is </p>
<p>$F_t=\frac{1}{2N}+(1-\frac{1}{2N})F_{t-1}$ .  </p>
<p>But either of the 2 autozygous alleles could mutate, so the probability that both will not mutate is $(1-\mu)^2$.  Hence,  we have</p>
<p>$F_t=(\frac{1}{2N}+(1-\frac{1}{2N})F_{t-1})*(1-\mu)^2$ .  </p>
<p>At equilibrium, $F_t=F_{t-1}$, this reduces to </p>
<p>$F_{eq}=\frac{(\frac{1}{2N})(1-\mu)^2}{1-(1-\frac{1}{2N})(1-\mu)^2} \approx \frac{1-2\mu}{2N(1-(1-\frac{1}{2N})(1-2\mu))}=\frac{1-2\mu}{1+4N\mu-2\mu} \approx \frac{1}{4N\mu+1}$ .  </p>
<p>Because $H=1-F_{eq}$ (heterozygosity, probability of non-IBD), then $H=\frac{4N\mu}{4N\mu+1}$. It implies that $H$ decreases as $\mu$ or $N$ decrease. Also, drift causes allele frequencies to fluctuate randomly, and these fluctuations occur more rapidly and drastically in smaller populations. If a population becomes separated into 2 populations, the variance between the populations increases due to drift.  </p>
<p>The key number is $4N\mu$ vs. $1$. When $4N\mu &lt; 1$, the stationary distribution of alleles bowl-shaped (fixed as either extreme); when $4N\mu &gt; 1$, the stationary distribution of alleles clumped in middle (variance maintained).   </p>
<p>Further, $H=\frac{4N\mu}{4N\mu+1}=\frac{\theta}{1+\theta}$, and $4N\mu=\theta$ becomes the basic level of variation, in which $\theta$ is population mutation rate. </p>
<blockquote>
<p>Why does $\theta=4N\mu$ ?</p>
<p>Estimating nucleotide divergence between 2 sequences as $\theta$, $\theta=\mu T*2=2\mu * 2N=4N\mu$, where $T=2N$ is coalescence time. See the <code>Coalescence effective population size</code> section for more details. </p>
<p>Meanwhile, we can re-write $H=\frac{P(mutation)}{P(mutation)+P(coalescence)}=\frac{2\mu}{2\mu+\frac{1}{2N}}=\frac{4Nu}{4Nu+1}$</p>
<p>For autosomal locus, $\theta=4N\mu$ (diploid, inherited by 2 parents)</p>
<p>For X chromosome, $\theta=3N\mu$ (diploid in females, haploid in males) </p>
<p>For Y and mtDNA, $\theta=N\mu$ (haploid, inherited uniparentally)  </p>
</blockquote>
<blockquote>
<p>Besides, gene flow is another way to introduce new alleles into the population. Replace $\mu$ with $m$ (where $m$ is the probability that a migrant contributes to the next generation, and $m$ is small), we have $F_{eq}=\frac{1}{1+4Nm}$.   </p>
</blockquote>
<h2 id="Estimates-of-effective-population-size-N-e"><a href="#Estimates-of-effective-population-size-N-e" class="headerlink" title="Estimates of effective population size ($N_e$)"></a>Estimates of effective population size ($N_e$)</h2><h3 id="Variations-in-population-size"><a href="#Variations-in-population-size" class="headerlink" title="Variations in population size"></a>Variations in population size</h3><p>Suppose that the population size in 2 successive generations are $N_0$ and $N_1$. </p>
<p>$F_2 = \frac{1}{2N_1}+(1-\frac{1}{2N_1})F_1$ ==&gt; $1-F_2 = (1-\frac{1}{2N_1})*(1-F_1)$</p>
<p>$1-F_1 = (1-\frac{1}{2N_0})*(1-F_0)$</p>
<p>These lead to  </p>
<p>$1-F_2=(1- \frac{1}{2N_1}) * (1- \frac{1}{2N_0}) * (1-F_0)$  </p>
<p>$1-F_t=(1- \frac{1}{2N_{t-1}}) * \dots * (1- \frac{1}{2N_1}) * (1- \frac{1}{2N_0}) * (1-F_0)$ , </p>
<p>by analogy with the constant $N$ case (where $N$ is now the effective population size, symbolized as $N_e$), it is appropriate to try to express this equation in the general form </p>
<p>$1-F_t = (1-\frac{1}{2N_e})^t*(1-F_0)$</p>
<p>setting the 2 expressions for $1-F_t$ equal to each other, we obtain  </p>
<p>$(1- \frac{1}{2N_e})^t=(1- \frac{1}{2N_{t-1}}) * \dots * (1- \frac{1}{2N_1}) * (1- \frac{1}{2N_0})$  </p>
<p>$\frac{1}{N_e} \approx \frac{1}{t}*(\frac{1}{N_0}+\frac{1}{N_1}+\dots+\frac{1}{N_{t-1}})$  </p>
<p>so the effective population size $N_e$ is the harmonic mean of the actual numbers. A single period of small population size (bottleneck) can result in a serious loss in heterozygosity. </p>
<h3 id="Variance-effective-population-size"><a href="#Variance-effective-population-size" class="headerlink" title="Variance effective population size"></a>Variance effective population size</h3><p>The variance effective population size $N_e^{(v)}$ is defined as the size of an idealized population with an given variance. Let $D(p)$ be the variance calculated for the actual population, $D(p)=\frac{p(1-p)}{2N}$. Then substitute $\hat D(p)$ for $D(p)$ and solve for $N$ which gives $N_e^{(v)}=\frac{p(1-p)}{2\hat D(p)}$. </p>
<p>See the <code>Unequal sex ratio</code> section for example. </p>
<h3 id="Inbreeding-effective-population-size"><a href="#Inbreeding-effective-population-size" class="headerlink" title="Inbreeding effective population size"></a>Inbreeding effective population size</h3><p>We define the inbreedinf effective population size $N_e^{(f)}$ as the size of the idealized population that has the same change in average inbreeding coefficient as the population under consideration. </p>
<p>We have the inbreeding coefficient ($F$), </p>
<p>$1-F_{t+1} = (1-\frac{1}{2N})*(1-F_t)$ . </p>
<p>Then the inbreeding effective size can be solved, </p>
<p>$N_e^{(f)}=\frac{1}{2(1-\frac{1-F_{t+1}}{1-F_t})}$ .   </p>
<h3 id="Coalescence-effective-population-size"><a href="#Coalescence-effective-population-size" class="headerlink" title="Coalescence effective population size"></a>Coalescence effective population size</h3><p>In a population with a constant effective population size with $2N_e$ copies of each locus, the propabbility that two alleles originate from the same parental copy (IBD) is $\frac{1}{2N_e}$. At each successive preceding generation, the probability of coalescence is the probability of non-coalescence at the $t-1$ preceding generations multiplied by the probability of coalescence at the generation of interest, </p>
<p>$P_c(t)=(1-\frac{1}{2N_e})^{t-1}(\frac{1}{2N_e})$, </p>
<p>which is geometrically distributed.  </p>
<p>For sufficiently large values of $N_e$, </p>
<p>$P_e(t) \approx \frac{1}{2N_e}e^{-\frac{t-1}{2N_e}}$. </p>
<p>Under the exponential distribution, both the expected value and the standard deviation equal to $2N_e$, which means that the expected time to coalescence is $2N_e$. </p>
<h3 id="Unequal-sex-ratio"><a href="#Unequal-sex-ratio" class="headerlink" title="Unequal sex ratio"></a>Unequal sex ratio</h3><p>If a sexual population consists of $N_m$ males and $N_f$ females, the actual size is $N_a=N_m+N_f$, but the effective population size for autosome- and X-linked gene is $N_e^A=\frac{4N_mN_f}{N_m+N_f}$ and $N_e^X=\frac{9N_mN_f}{4N_m+2N_f}$, respectively.  </p>
<p>Note that the sampling variance for the X chromosomes from males is $\frac{p_mq_m}{N_m}$, whereasthe sampling variance for X chromosomes from female is $\frac{p_fq_f}{2N_f}$, in which $p_m$ and $p_f$ are allele frequencies respectively in males and females. </p>
<blockquote>
<p>Since $\frac{2}{3}$ of the X chromosomes in any generation come from females in the previous generation and $\frac{1}{3}$ come from males. </p>
<p>For a given generation, half its autosomes must be contributed by females in the previous generation and the other half from males.   </p>
<p>Under the binomial distribution, we have $E=Np$ and $D=Npq$, where $E$ and $D$ respectively denote mean and variance and $N$ is sample size (# individual). For any male or female individuals, $E=E(\frac{\sum{p}}{N})=p$ and $D=D(\frac{\sum{p(1-p)}}{N})=\frac{Npq}{N^2}=\frac{pq}{N}$, where $N$ indicates the number of sequence ($N=N_m$ and $N=2N_f$). </p>
</blockquote>
<p>The allele frequency in the population is $p=\frac{1}{3}p_m+\frac{2}{3}p_f$. The variance of $p$ is $D(p)=\frac{1}{9}(\frac{p_mq_m}{N_m})+\frac{4}{9}(\frac{p_fq_f}{2N_f})$. </p>
<blockquote>
<p>Because $p_m$ and $p_f$ are independent random variables, $D(p)=\frac{1}{9}D(p_m)+\frac{4}{9}D(p_f)$.  </p>
</blockquote>
<p>As steady state, $p_m=p_f=p$ and $q_m=q_f=q$. </p>
<p>$D(p)=pq(\frac{1}{9}\frac{1}{N_m}+\frac{4}{9}\frac{1}{2N_f})=\frac{pq}{2(\frac{9N_mN_f}{4N_m+2N_f})}=\frac{pq}{2N_e}$. </p>
<p>It’s a variance effective size, and the binomal sampling variance in an ideal population is $\frac{pq}{2N_e}$.  </p>
<hr>
<p>The end. </p>
<p>Have a Nice Day! </p>

    </div>

    
    
    
        <div class="reward-container">
  <div>Peace & Love</div>
  <button onclick="var qr = document.getElementById('qr'); qr.style.display = (qr.style.display === 'none') ? 'block' : 'none';">
    Donate
  </button>
  <div id="qr" style="display: none;">
      
      <div style="display: inline-block;">
        <img src="/images/wechat.JPG" alt="KNOCK KNOCK KNOCKIN WeChat Pay">
        <p>WeChat Pay</p>
      </div>
      
      <div style="display: inline-block;">
        <img src="/images/ali.JPG" alt="KNOCK KNOCK KNOCKIN Alipay">
        <p>Alipay</p>
      </div>

  </div>
</div>

        

<div>
<ul class="post-copyright">
  <li class="post-copyright-author">
    <strong>Post author:  </strong>KNOCK KNOCK KNOCKIN
  </li>
  <li class="post-copyright-link">
    <strong>Post link: </strong>
    <a href="https://panyuwen.github.io/2020/08/21/7-effective-population-size/" title="7. effective population size">https://panyuwen.github.io/2020/08/21/7-effective-population-size/</a>
  </li>
  <li class="post-copyright-license">
    <strong>Copyright Notice:  </strong>All articles in this blog are licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" rel="noopener" target="_blank"><i class="fab fa-fw fa-creative-commons"></i>BY-NC-SA</a> unless stating additionally.
  </li>
</ul>
</div>


      <footer class="post-footer">
          <div class="post-tags">
              <a href="/tags/population-genetics/" rel="tag"># population genetics</a>
              <a href="/tags/diversity/" rel="tag"># diversity</a>
          </div>

        


        
    <div class="post-nav">
      <div class="post-nav-item">
    <a href="/2020/08/06/6-genetic-diversity/" rel="prev" title="6. genetic diversity">
      <i class="fa fa-chevron-left"></i> 6. genetic diversity
    </a></div>
      <div class="post-nav-item">
    <a href="/2020/08/24/8-blowin-in-the-wind/" rel="next" title="8. blowin' in the wind">
      8. blowin' in the wind <i class="fa fa-chevron-right"></i>
    </a></div>
    </div>
      </footer>
    
  </article>
  
  
  



          </div>
          

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          Table of Contents
        </li>
        <li class="sidebar-nav-overview">
          Overview
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
          <div class="post-toc motion-element"><ol class="nav"><li class="nav-item nav-level-1"><a class="nav-link" href="#Effective-population-size-N-e"><span class="nav-number">1.</span> <span class="nav-text">Effective population size ($N_e$)</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#Preface"><span class="nav-number">1.1.</span> <span class="nav-text">Preface</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Fixation-index"><span class="nav-number">1.2.</span> <span class="nav-text">Fixation index</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#The-mutation-drift-balance"><span class="nav-number">1.3.</span> <span class="nav-text">The mutation-drift balance</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Estimates-of-effective-population-size-N-e"><span class="nav-number">1.4.</span> <span class="nav-text">Estimates of effective population size ($N_e$)</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Variations-in-population-size"><span class="nav-number">1.4.1.</span> <span class="nav-text">Variations in population size</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Variance-effective-population-size"><span class="nav-number">1.4.2.</span> <span class="nav-text">Variance effective population size</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Inbreeding-effective-population-size"><span class="nav-number">1.4.3.</span> <span class="nav-text">Inbreeding effective population size</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Coalescence-effective-population-size"><span class="nav-number">1.4.4.</span> <span class="nav-text">Coalescence effective population size</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Unequal-sex-ratio"><span class="nav-number">1.4.5.</span> <span class="nav-text">Unequal sex ratio</span></a></li></ol></li></ol></li></ol></div>
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
    <img class="site-author-image" itemprop="image" alt="KNOCK KNOCK KNOCKIN"
      src="/images/IMG_F7DE10EB3810-1.jpeg">
  <p class="site-author-name" itemprop="name">KNOCK KNOCK KNOCKIN</p>
  <div class="site-description" itemprop="description">life, study, and work</div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives/">
        
          <span class="site-state-item-count">8</span>
          <span class="site-state-item-name">posts</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
            <a href="/categories/">
        <span class="site-state-item-count">3</span>
        <span class="site-state-item-name">categories</span></a>
      </div>
      <div class="site-state-item site-state-tags">
            <a href="/tags/">
        <span class="site-state-item-count">11</span>
        <span class="site-state-item-name">tags</span></a>
      </div>
  </nav>
</div>
  <div class="links-of-author motion-element">
      <span class="links-of-author-item">
        <a href="https://github.com/panyuwen" title="GitHub → https:&#x2F;&#x2F;github.com&#x2F;panyuwen" rel="noopener" target="_blank"><i class="fab fa-github fa-fw"></i>GitHub</a>
      </span>
      <span class="links-of-author-item">
        <a href="mailto:panyuwen.x@gmail.com" title="E-Mail → mailto:panyuwen.x@gmail.com" rel="noopener" target="_blank"><i class="fa fa-envelope fa-fw"></i>E-Mail</a>
      </span>
  </div>
  <div class="cc-license motion-element" itemprop="license">
    <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" class="cc-opacity" rel="noopener" target="_blank"><img src="/images/cc-by-nc-sa.svg" alt="Creative Commons"></a>
  </div>



      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<div class="copyright">
  
  &copy; 
  <span itemprop="copyrightYear">2020</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">KNOCK KNOCK KNOCKIN</span>
</div>

        
<div class="busuanzi-count">
  <script async src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
    <span class="post-meta-item" id="busuanzi_container_site_uv" style="display: none;">
      <span class="post-meta-item-icon">
        <i class="fa fa-user"></i>
      </span>
      <span class="site-uv" title="Total Visitors">
        <span id="busuanzi_value_site_uv"></span>
      </span>
    </span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item" id="busuanzi_container_site_pv" style="display: none;">
      <span class="post-meta-item-icon">
        <i class="fa fa-eye"></i>
      </span>
      <span class="site-pv" title="Total Views">
        <span id="busuanzi_value_site_pv"></span>
      </span>
    </span>
</div>








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/muse.js"></script>


<script src="/js/next-boot.js"></script>

<script src="/js/bookmark.js"></script>




  




  
<script src="/js/local-search.js"></script>













  

  
      

<script>
  if (typeof MathJax === 'undefined') {
    window.MathJax = {
      loader: {
        source: {
          '[tex]/amsCd': '[tex]/amscd',
          '[tex]/AMScd': '[tex]/amscd'
        }
      },
      tex: {
        inlineMath: {'[+]': [['$', '$']]},
        tags: 'ams'
      },
      options: {
        renderActions: {
          findScript: [10, doc => {
            document.querySelectorAll('script[type^="math/tex"]').forEach(node => {
              const display = !!node.type.match(/; *mode=display/);
              const math = new doc.options.MathItem(node.textContent, doc.inputJax[0], display);
              const text = document.createTextNode('');
              node.parentNode.replaceChild(text, node);
              math.start = {node: text, delim: '', n: 0};
              math.end = {node: text, delim: '', n: 0};
              doc.math.push(math);
            });
          }, '', false],
          insertedScript: [200, () => {
            document.querySelectorAll('mjx-container').forEach(node => {
              let target = node.parentNode;
              if (target.nodeName.toLowerCase() === 'li') {
                target.parentNode.classList.add('has-jax');
              }
            });
          }, '', false]
        }
      }
    };
    (function () {
      var script = document.createElement('script');
      script.src = '//cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js';
      script.defer = true;
      document.head.appendChild(script);
    })();
  } else {
    MathJax.startup.document.state(0);
    MathJax.texReset();
    MathJax.typeset();
  }
</script><!-- hexo-inject:begin --><!-- hexo-inject:end -->

    

  

</body>
</html>
