<!DOCTYPE html>
<html lang="en">
<head>
  <!-- hexo-inject:begin --><!-- hexo-inject:end --><meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 5.0.0">
  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32-next.png">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16-next.png">
  <link rel="mask-icon" href="/images/logo.svg" color="#222">

<link rel="stylesheet" href="/css/main.css">


<link rel="stylesheet" href="/lib/font-awesome/css/all.min.css">

<script id="hexo-configurations">
    var NexT = window.NexT || {};
    var CONFIG = {"hostname":"panyuwen.github.io","root":"/","scheme":"Muse","version":"7.8.0","exturl":false,"sidebar":{"position":"left","display":"always","padding":18,"offset":12,"onmobile":false},"copycode":{"enable":true,"show_result":false,"style":"mac"},"back2top":{"enable":true,"sidebar":false,"scrollpercent":true},"bookmark":{"enable":true,"color":"#222","save":"manual"},"fancybox":false,"mediumzoom":false,"lazyload":false,"pangu":false,"comments":{"style":"tabs","active":null,"storage":true,"lazyload":false,"nav":null},"algolia":{"hits":{"per_page":10},"labels":{"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}},"localsearch":{"enable":true,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},"motion":{"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}},"path":"search.xml"};
  </script>

  <meta name="description" content="Genetic diversityPrefaceGenetic diveristy describes the genetic differences among individuals within the same population. The higher genetic diveristy may imply the larger gene pool, and also indicate">
<meta property="og:type" content="article">
<meta property="og:title" content="6. genetic diversity">
<meta property="og:url" content="https://panyuwen.github.io/2020/08/06/6-genetic-diversity/index.html">
<meta property="og:site_name" content="yuwen pan&#39;s blog">
<meta property="og:description" content="Genetic diversityPrefaceGenetic diveristy describes the genetic differences among individuals within the same population. The higher genetic diveristy may imply the larger gene pool, and also indicate">
<meta property="og:locale" content="en_US">
<meta property="article:published_time" content="2020-08-06T03:35:13.000Z">
<meta property="article:modified_time" content="2020-08-11T02:09:15.787Z">
<meta property="article:author" content="yuwen pan">
<meta property="article:tag" content="population genetics">
<meta property="article:tag" content="diversity">
<meta name="twitter:card" content="summary">

<link rel="canonical" href="https://panyuwen.github.io/2020/08/06/6-genetic-diversity/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome : false,
    isPost : true,
    lang   : 'en'
  };
</script>

  <title>6. genetic diversity | yuwen pan's blog</title>
  






  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript><!-- hexo-inject:begin --><!-- hexo-inject:end -->

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <!-- hexo-inject:begin --><!-- hexo-inject:end --><div class="container use-motion">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-nav-toggle">
    <div class="toggle" aria-label="Toggle navigation bar">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>

  <div class="site-meta">

    <a href="/" class="brand" rel="start">
      <span class="logo-line-before"><i></i></span>
      <h1 class="site-title">yuwen pan's blog</h1>
      <span class="logo-line-after"><i></i></span>
    </a>
      <p class="site-subtitle" itemprop="description">life, study, and work</p>
  </div>

  <div class="site-nav-right">
    <div class="toggle popup-trigger">
        <i class="fa fa-search fa-fw fa-lg"></i>
    </div>
  </div>
</div>




<nav class="site-nav">
  <ul id="menu" class="main-menu menu">
        <li class="menu-item menu-item-home">

    <a href="/" rel="section"><i class="fa fa-home fa-fw"></i>Home</a>

  </li>
        <li class="menu-item menu-item-archives">

    <a href="/archives/" rel="section"><i class="fa fa-archive fa-fw"></i>Archives</a>

  </li>
      <li class="menu-item menu-item-search">
        <a role="button" class="popup-trigger"><i class="fa fa-search fa-fw"></i>Search
        </a>
      </li>
  </ul>
</nav>



  <div class="search-pop-overlay">
    <div class="popup search-popup">
        <div class="search-header">
  <span class="search-icon">
    <i class="fa fa-search"></i>
  </span>
  <div class="search-input-container">
    <input autocomplete="off" autocapitalize="off"
           placeholder="Searching..." spellcheck="false"
           type="search" class="search-input">
  </div>
  <span class="popup-btn-close">
    <i class="fa fa-times-circle"></i>
  </span>
</div>
<div id="search-result">
  <div id="no-result">
    <i class="fa fa-spinner fa-pulse fa-5x fa-fw"></i>
  </div>
</div>

    </div>
  </div>

</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>
  <a role="button" class="book-mark-link book-mark-link-fixed"></a>

  <a href="https://github.com/panyuwen" class="github-corner" title="Follow me" aria-label="Follow me" rel="noopener" target="_blank"><svg width="80" height="80" viewBox="0 0 250 250" aria-hidden="true"><path d="M0,0 L115,115 L130,115 L142,142 L250,250 L250,0 Z"></path><path d="M128.3,109.0 C113.8,99.7 119.0,89.6 119.0,89.6 C122.0,82.7 120.5,78.6 120.5,78.6 C119.2,72.0 123.4,76.3 123.4,76.3 C127.3,80.9 125.5,87.3 125.5,87.3 C122.9,97.6 130.6,101.9 134.4,103.2" fill="currentColor" style="transform-origin: 130px 106px;" class="octo-arm"></path><path d="M115.0,115.0 C114.9,115.1 118.7,116.5 119.8,115.4 L133.7,101.6 C136.9,99.2 139.9,98.4 142.2,98.6 C133.8,88.0 127.5,74.4 143.8,58.0 C148.5,53.4 154.0,51.2 159.7,51.0 C160.3,49.4 163.2,43.6 171.4,40.1 C171.4,40.1 176.1,42.5 178.8,56.2 C183.1,58.6 187.2,61.8 190.9,65.4 C194.5,69.0 197.7,73.2 200.1,77.6 C213.8,80.2 216.3,84.9 216.3,84.9 C212.7,93.1 206.9,96.0 205.4,96.6 C205.1,102.4 203.0,107.8 198.3,112.5 C181.9,128.9 168.3,122.5 157.7,114.1 C157.9,116.9 156.7,120.9 152.7,124.9 L141.0,136.5 C139.8,137.7 141.6,141.9 141.8,141.8 Z" fill="currentColor" class="octo-body"></path></svg></a>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          

          <div class="content post posts-expand">
            

    
  
  
  <article itemscope itemtype="http://schema.org/Article" class="post-block" lang="en">
    <link itemprop="mainEntityOfPage" href="https://panyuwen.github.io/2020/08/06/6-genetic-diversity/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="image" content="/images/IMG_F7DE10EB3810-1.jpeg">
      <meta itemprop="name" content="yuwen pan">
      <meta itemprop="description" content="to share my ideas and codes">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="yuwen pan's blog">
    </span>
      <header class="post-header">
        <h1 class="post-title" itemprop="name headline">
          6. genetic diversity
        </h1>

        <div class="post-meta">
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-calendar"></i>
              </span>
              <span class="post-meta-item-text">Posted on</span>

              <time title="Created: 2020-08-06 11:35:13" itemprop="dateCreated datePublished" datetime="2020-08-06T11:35:13+08:00">2020-08-06</time>
            </span>
              <span class="post-meta-item">
                <span class="post-meta-item-icon">
                  <i class="far fa-calendar-check"></i>
                </span>
                <span class="post-meta-item-text">Edited on</span>
                <time title="Modified: 2020-08-11 10:09:15" itemprop="dateModified" datetime="2020-08-11T10:09:15+08:00">2020-08-11</time>
              </span>
            <span class="post-meta-item">
              <span class="post-meta-item-icon">
                <i class="far fa-folder"></i>
              </span>
              <span class="post-meta-item-text">In</span>
                <span itemprop="about" itemscope itemtype="http://schema.org/Thing">
                  <a href="/categories/statistics/" itemprop="url" rel="index"><span itemprop="name">statistics</span></a>
                </span>
            </span>

          
            <span class="post-meta-item" title="Views" id="busuanzi_container_page_pv" style="display: none;">
              <span class="post-meta-item-icon">
                <i class="fa fa-eye"></i>
              </span>
              <span class="post-meta-item-text">Views: </span>
              <span id="busuanzi_value_page_pv"></span>
            </span>

        </div>
      </header>

    
    
    
    <div class="post-body" itemprop="articleBody">

      
        <h1 id="Genetic-diversity"><a href="#Genetic-diversity" class="headerlink" title="Genetic diversity"></a>Genetic diversity</h1><h2 id="Preface"><a href="#Preface" class="headerlink" title="Preface"></a>Preface</h2><p>Genetic diveristy describes the genetic differences among individuals within the same population. The higher genetic diveristy may imply the larger gene pool, and also indicate the higher stability. </p>
<a id="more"></a>

<p>Factors including genetic drift, natural selection, migration, gene flow, population admixture, bottleneck, and sexual selection may influence the genetic diveristy. </p>
<p>In the last ~10 thousand years, since the human dispersal out of Afria, our ancestors had experienced war, hunger, and all kinds of changing environment. They survived, the way out of Africa made them stronger. All of these experiences were recorded in our blood, in every nucleotide on our genome. </p>
<p>We have to know about ourselves, about our origin, also about our futher. The key to the futher is on our genome, passed from our ancestors. We are generating huge mount of genetic data and we are decoding the variations on our genome. We are amazed about the diversity of our genome. </p>
<p>Now, let’s focus on the etimators of the genetic diveristy. </p>
<p>Before we start, you can also go through the introduction to <a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Genetic_diversity">genetic diveristy</a> on Wikipedia</p>
<h2 id="DNA-polymorphism"><a href="#DNA-polymorphism" class="headerlink" title="DNA polymorphism"></a>DNA polymorphism</h2><p>DNA polymorphisms indicates the variations on DNA sequences among individuals and populations. The forms of DNA polymorphisms include <code>single nucleotide polymorphisms (SNPs)</code>, variable number of tandem repeats (VNTRs, e.g., minisatellites), short tandom repeat (STR, e.g., microsatellite), copy number variation (CNV), and so on. </p>
<p>SNPs are the most common type of genetic variations in humans (90% of all human DNA polymorphism). A SNP is a substitution of a single nucleotide at a specific position in the genome, that is present in a sufficiently large fraction of the population (e.g. 1% or more). And a single-nucleotide variant (SNV) is a variation in a single nucleotide without any <code>limitations of frequency </code>(which should be minor allele frequency (MAF) &gt;1%). </p>
<p>There has been identified millions of comman SNPs among human populations (MAF &gt;5%) (roughly, say 5~10 million). It implies ~0.1% difference between every two humans. P.S. it should be ~1% between human and Chimpanzee. </p>
<blockquote>
<p><a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism">https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism</a><br>Dr. Shuhua Xu’s slides</p>
</blockquote>
<p>In population genetics, we usually focus on biallelic SNPs, which means two allelic variants are segregating in the population. Since majority of SNPs in humans are biallelic. It’s the low mutation rate and short evolution time of human beings that matters, time is required to have mutations occured on human genome, and much more time is required to have mutations occured on the same locus more than once.</p>
<blockquote>
<p>mutation rate: $\mu=1.25*10^{-8}$ per site per generation<br>time to most comman ancestor (TMRCA): ~13 million years between human &amp; chimpanzee</p>
<p>mutation rate ($\mu$) would be $2.5*10^{-8}$ per site per generation if TMRCA=~6 million years</p>
<p>so we have 2 mutation rates ($\mu$), slow &amp; fast</p>
</blockquote>
<h2 id="Heterozygosity"><a href="#Heterozygosity" class="headerlink" title="Heterozygosity"></a>Heterozygosity</h2><p>The fraction of individuals in a population that are heterozygous for a particular locus.</p>
<p>Observed: $H_o=\frac{\sum_{1}^{n}(1 \ if \ a_{i1}\ne a_{i2})}{n}$</p>
<blockquote>
<p>where n is the number of individuals in the population, and $a_{i1}$, $a_{i2}$ are the alleles of individual i at the target locus.</p>
</blockquote>
<p>Expected: $H_e=1-\sum_{1}^{m}(f_i)^2$</p>
<blockquote>
<p>where m is the number of alleles at the target locus, and $f_i$ is the allele frequency of the $i^{th}$ allele at the target locus.</p>
</blockquote>
<p>High heterozygosity means lots of genetic variability.  Low heterozygosity means little genetic variability, which may also indicate severe effects of small population sizes (e.g., population bottlenecks). </p>
<p>Often, we will compare the observed level of heterozygosity to what we expect under Hardy-Weinberg equilibrium (HWE). If the observed heterozygosity is lower than expected, we seek to attribute the discrepancy to forces such as inbreeding. If heterozygosity is higher than expected, we might suspect an isolate-breaking effect (the mixing of two previously isolated populations). </p>
<blockquote>
<p>HWE: $p^2+2pq+q^2=1$ under the <code>Wright-Fisher Model</code></p>
<p>Wright-Fisher model: </p>
<blockquote>
<ul>
<li>Infinite population size</li>
<li>Non-overlapping generation</li>
<li>Random mating</li>
<li>No natural selection</li>
<li>…</li>
</ul>
</blockquote>
<p><a target="_blank" rel="noopener" href="http://www.uwyo.edu/dbmcd/molmark/lect04/lect4.html">http://www.uwyo.edu/dbmcd/molmark/lect04/lect4.html</a><br>Dr. Shuhua Xu’s slides</p>
</blockquote>
<blockquote>
<p>We can further calcluate the F statistic based on heterozygosity.  </p>
</blockquote>
<h2 id="Effective-population-size"><a href="#Effective-population-size" class="headerlink" title="Effective population size"></a>Effective population size</h2><p>To be finished…</p>
<p>We are going to discuss about the <code>F statistic</code>.</p>
<h2 id="Population-mutation-rate-theta"><a href="#Population-mutation-rate-theta" class="headerlink" title="Population mutation rate ($\theta$)"></a>Population mutation rate ($\theta$)</h2><p>In population genetics, population mutation rate ($\theta$) describe the genetic diversity in a population.  </p>
<ul>
<li>$\theta=4N_e\mu$ for autosome</li>
<li>$\theta=N_e\mu$ for Y and mtDNA</li>
<li>$\theta=3N_e\mu$ for X chromosome</li>
</ul>
<blockquote>
<p>where $N_e$ is the effective population size and $\mu$  is the per-generation mutation rate of the population of interest</p>
<p>see the <code>Effective population size</code> section for more details about these formulas</p>
</blockquote>
<p>There are many estimators of $\theta$, including average pairwise differences (nucleotide diversity) ($\theta_\pi$), number of segregating sites ($\theta_K$), and so on.</p>
<h3 id="Nucleotide-diversity-theta-pi"><a href="#Nucleotide-diversity-theta-pi" class="headerlink" title="Nucleotide diversity ($\theta_\pi$)"></a><a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Nucleotide_diversity">Nucleotide diversity</a> ($\theta_\pi$)</h3><p>One commonly used measure of nucleotide diversity was first introduced by Nei and Li in 1979. This measure is defined as the average number of nucleotide differences per site between two DNA sequences in all possible pairs in the sample population, and is denoted by $\pi$.</p>
<p>It is given by the formula:</p>
<p>$\pi=\sum{x_ix_j\pi_{ij}}=2*\sum_{i=2}^{n}{\sum_{j=1}^{i-1}x_ix_j\pi_{ij}}$</p>
<blockquote>
<p>where $x_i$ and $x_j$ are the respective frequencies of the $i^{th}$ and $j^{th}$ sequences, $\pi_{ij}$ is the number of nucleotide differences per nucleotide site between the $i^{th}$ and $j^{th}$ sequences, and $n$ is the number of sequences in the sample.</p>
</blockquote>
<blockquote>
<p>Nei, M., and Li, W.H. (1979). MATHEMATICAL-MODEL FOR STUDYING GENETIC-VARIATION IN TERMS OF RESTRICTION ENDONUCLEASES. PANS</p>
</blockquote>
<p>There is an easier way to calculate $\pi$.<br>For a particular locus $i$ in one population, we have</p>
<p>$\pi_i=\frac{number \ of \ allele \ 1 \ * \  number \ of \ allele \ 2}{n*(n-1)/2}$</p>
<blockquote>
<p>for example:</p>
<blockquote>
<p>sequence 1: NNNNNNNNNN<code>A</code>NNNNNNNNNN</p>
<p>sequence 2: NNNNNNNNNN<code>C</code>NNNNNNNNNN</p>
<p>sequence 3: NNNNNNNNNN<code>A</code>NNNNNNNNNN</p>
</blockquote>
<p>allele A count = 2, allele C count = 1</p>
<p>total pairwise difference(s) = $2*1$</p>
<p>number of alll possible pairs = $3*(3-1)/2$</p>
<p>$\pi=0.67$</p>
</blockquote>
<p>then, $\pi=\sum_{}^{L}\pi_i$</p>
<blockquote>
<p>where $n$ is the number of sequences, and $L$ is the total number of loci on the DNA sequence</p>
<p>which means we can easily calculate $\pi$ based on the summary data of allele frequency</p>
<p>this formula is only for biallelic loci. It’s also easy to conduct the formula for multiallelic loci.</p>
</blockquote>
<p>In general, $\pi_{AFR}&gt;\pi_{EUR}&gt;\pi_{EAS}$, $\pi_{admixed}&gt;\pi_{reference}$</p>
<h3 id="Number-of-segregating-site-theta-K"><a href="#Number-of-segregating-site-theta-K" class="headerlink" title="Number of segregating site ($\theta_K$)"></a>Number of segregating site ($\theta_K$)</h3><p>to be finished…</p>
<h2 id="Tajima’s-D"><a href="#Tajima’s-D" class="headerlink" title="Tajima’s D"></a>Tajima’s D</h2><h2 id="Haplotype-diversity"><a href="#Haplotype-diversity" class="headerlink" title="Haplotype diversity"></a>Haplotype diversity</h2><p>to be finished…</p>
<h2 id="Time-to-Most-Recent-Common-Ancestor-TMRCA"><a href="#Time-to-Most-Recent-Common-Ancestor-TMRCA" class="headerlink" title="Time to Most Recent Common Ancestor (TMRCA)"></a><a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Most_recent_common_ancestor">Time to Most Recent Common Ancestor (TMRCA)</a></h2><p>It’s a long story, starting from the very begining. God created the world as well ancestors of humans, Adam and Eve. Then, Adam and Eve created human beings.  </p>
<p>OK, let’s talk about science. A lot of work have been done to build and improve the human Y chromosome and mitochondrial phylogenetic tree. Since mitochondrial DNA (mtDNA) and Y chromosome are respectively inherited maternally and paternally, while recombination is rare or absent, it seems relatively easy to track the ancestry of the lineages back to a Most Recent Common Ancestor (MRCA). And scientists have helped to trace our lineages back to single person living in Africa, named <a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Mitochondrial_Eve">Mitochondrial Eve</a> and <a target="_blank" rel="noopener" href="https://en.wikipedia.org/wiki/Y-chromosomal_Adam">Y-chromosomal Adam</a>. The principal is the coalescence of mutations on every pairs of sequences. Meanwhile, we are also able to calculate TMRCA by estimating the expected time to have all of the observed mutations occured on Y chromosome / mtDNA. </p>
<p>For autosome, there is also a simple way to estimate TMCRA. First, chose a short segment on the genome to avoid potential recombination. Then calculate the average pairwise differences among sequences (in the same way as $\pi$).  </p>
<blockquote>
<p>not necessarily sequences from the same population, depend on your purpose. you are able to estimate the within population TMRCA using sequences from the same population. or you can estimate the cross-population TMRCA (divergence time) using sequences from different populations.  </p>
<p>formulas would be a little bit different  </p>
<p>within-population:<br>$\pi=\sum{\frac{number \ of \ allele \ 1 \ * \  number \ of \ allele \ 2}{n*(n-1)/2}}$</p>
<p>cross-population:<br>$\pi=\sum{\frac{c_{1A} \ * \ c_{2B} \ + \ c_{2A} \ * \ c_{1B}}{n_A \ * \ n_B}}$  </p>
<p>$c_{1A}$: number of allele 1 in population A<br>$c_{2A}$: number of allele 2 in population A<br>$c_{1B}$: number of allele 1 in population B<br>$c_{2B}$: number of allele 2 in population B  </p>
</blockquote>
<p>$TMRCA=\frac{\pi}{2 \mu L}$  </p>
<blockquote>
<p>where $\mu$ denotes the local mutation rate, and $L$ is the length of the regional segment.  </p>
</blockquote>
<p>$\mu=\frac{d_{human \ vs. \ human-ape \ ancestor}}{L \ * \ T_{human \ to \ human-ape \ ancestor}}$  </p>
<p>then,  </p>
<p>$TMRCA=\frac{T_{human \ to \ human-ape \ ancestor} \ * \ \pi}{d_{human \ vs. \ human-ape \ ancestor} \ * \ 2}$</p>
<blockquote>
<p>$d_{human \ vs. \ human-ape \ ancestor}$ is the nucleotide difference of the target region between the human reference genome and that of the most recent common ancestor of humans-chimpanzee. ($\pi$ of 2 sequences)</p>
<p>$d_{human \ vs. \ human-ape \ ancestor}$ should be half the difference between the human reference genome and that of the chimpanzee </p>
<p>$T_{human \ to \ human-ape \ ancestor}$ is the divergence time of humans and chimpanzees. Here we use 13 million years.<br>Lu, Dongsheng, Haiyi Lou, Kai Yuan, Xiaoji Wang, Yuchen Wang, Chao Zhang, Yan Lu et al. “Ancestral origins and genetic history of Tibetan highlanders.” The American Journal of Human Genetics (2016)</p>
</blockquote>
<p>Other ways to date genomic variants includes <a target="_blank" rel="noopener" href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3000586">GEVA</a></p>
<blockquote>
<p>Albers, Patrick K., and Gil McVean. “Dating genomic variants and shared ancestry in population-scale sequencing data.” PLoS biology (2020)</p>
</blockquote>

    </div>

    
    
    
        <div class="reward-container">
  <div>Peace & Love</div>
  <button onclick="var qr = document.getElementById('qr'); qr.style.display = (qr.style.display === 'none') ? 'block' : 'none';">
    Donate
  </button>
  <div id="qr" style="display: none;">
      
      <div style="display: inline-block;">
        <img src="/images/wechat.JPG" alt="yuwen pan WeChat Pay">
        <p>WeChat Pay</p>
      </div>
      
      <div style="display: inline-block;">
        <img src="/images/ali.JPG" alt="yuwen pan Alipay">
        <p>Alipay</p>
      </div>

  </div>
</div>

        

<div>
<ul class="post-copyright">
  <li class="post-copyright-author">
    <strong>Post author:  </strong>yuwen pan
  </li>
  <li class="post-copyright-link">
    <strong>Post link: </strong>
    <a href="https://panyuwen.github.io/2020/08/06/6-genetic-diversity/" title="6. genetic diversity">https://panyuwen.github.io/2020/08/06/6-genetic-diversity/</a>
  </li>
  <li class="post-copyright-license">
    <strong>Copyright Notice:  </strong>All articles in this blog are licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" rel="noopener" target="_blank"><i class="fab fa-fw fa-creative-commons"></i>BY-NC-SA</a> unless stating additionally.
  </li>
</ul>
</div>


      <footer class="post-footer">
          <div class="post-tags">
              <a href="/tags/population-genetics/" rel="tag"># population genetics</a>
              <a href="/tags/diversity/" rel="tag"># diversity</a>
          </div>

        


        
    <div class="post-nav">
      <div class="post-nav-item">
    <a href="/2020/08/05/5-something-about-FDR/" rel="prev" title="5. something about FDR">
      <i class="fa fa-chevron-left"></i> 5. something about FDR
    </a></div>
      <div class="post-nav-item"></div>
    </div>
      </footer>
    
  </article>
  
  
  



          </div>
          

<script>
  window.addEventListener('tabs:register', () => {
    let { activeClass } = CONFIG.comments;
    if (CONFIG.comments.storage) {
      activeClass = localStorage.getItem('comments_active') || activeClass;
    }
    if (activeClass) {
      let activeTab = document.querySelector(`a[href="#comment-${activeClass}"]`);
      if (activeTab) {
        activeTab.click();
      }
    }
  });
  if (CONFIG.comments.storage) {
    window.addEventListener('tabs:click', event => {
      if (!event.target.matches('.tabs-comment .tab-content .tab-pane')) return;
      let commentClass = event.target.classList[1];
      localStorage.setItem('comments_active', commentClass);
    });
  }
</script>

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          Table of Contents
        </li>
        <li class="sidebar-nav-overview">
          Overview
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
          <div class="post-toc motion-element"><ol class="nav"><li class="nav-item nav-level-1"><a class="nav-link" href="#Genetic-diversity"><span class="nav-number">1.</span> <span class="nav-text">Genetic diversity</span></a><ol class="nav-child"><li class="nav-item nav-level-2"><a class="nav-link" href="#Preface"><span class="nav-number">1.1.</span> <span class="nav-text">Preface</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#DNA-polymorphism"><span class="nav-number">1.2.</span> <span class="nav-text">DNA polymorphism</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Heterozygosity"><span class="nav-number">1.3.</span> <span class="nav-text">Heterozygosity</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Effective-population-size"><span class="nav-number">1.4.</span> <span class="nav-text">Effective population size</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Population-mutation-rate-theta"><span class="nav-number">1.5.</span> <span class="nav-text">Population mutation rate ($\theta$)</span></a><ol class="nav-child"><li class="nav-item nav-level-3"><a class="nav-link" href="#Nucleotide-diversity-theta-pi"><span class="nav-number">1.5.1.</span> <span class="nav-text">Nucleotide diversity ($\theta_\pi$)</span></a></li><li class="nav-item nav-level-3"><a class="nav-link" href="#Number-of-segregating-site-theta-K"><span class="nav-number">1.5.2.</span> <span class="nav-text">Number of segregating site ($\theta_K$)</span></a></li></ol></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Tajima%E2%80%99s-D"><span class="nav-number">1.6.</span> <span class="nav-text">Tajima’s D</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Haplotype-diversity"><span class="nav-number">1.7.</span> <span class="nav-text">Haplotype diversity</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Time-to-Most-Recent-Common-Ancestor-TMRCA"><span class="nav-number">1.8.</span> <span class="nav-text">Time to Most Recent Common Ancestor (TMRCA)</span></a></li></ol></li></ol></div>
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
    <img class="site-author-image" itemprop="image" alt="yuwen pan"
      src="/images/IMG_F7DE10EB3810-1.jpeg">
  <p class="site-author-name" itemprop="name">yuwen pan</p>
  <div class="site-description" itemprop="description">to share my ideas and codes</div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
          <a href="/archives/">
        
          <span class="site-state-item-count">6</span>
          <span class="site-state-item-name">posts</span>
        </a>
      </div>
      <div class="site-state-item site-state-categories">
            <a href="/categories/">
        <span class="site-state-item-count">2</span>
        <span class="site-state-item-name">categories</span></a>
      </div>
      <div class="site-state-item site-state-tags">
            <a href="/tags/">
        <span class="site-state-item-count">12</span>
        <span class="site-state-item-name">tags</span></a>
      </div>
  </nav>
</div>
  <div class="links-of-author motion-element">
      <span class="links-of-author-item">
        <a href="https://github.com/panyuwen" title="GitHub → https:&#x2F;&#x2F;github.com&#x2F;panyuwen" rel="noopener" target="_blank"><i class="fab fa-github fa-fw"></i>GitHub</a>
      </span>
      <span class="links-of-author-item">
        <a href="mailto:panyuwen.x@gmail.com" title="E-Mail → mailto:panyuwen.x@gmail.com" rel="noopener" target="_blank"><i class="fa fa-envelope fa-fw"></i>E-Mail</a>
      </span>
  </div>
  <div class="cc-license motion-element" itemprop="license">
    <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" class="cc-opacity" rel="noopener" target="_blank"><img src="/images/cc-by-nc-sa.svg" alt="Creative Commons"></a>
  </div>



      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

        

<div class="copyright">
  
  &copy; 
  <span itemprop="copyrightYear">2020</span>
  <span class="with-love">
    <i class="fa fa-heart"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">yuwen pan</span>
</div>

        
<div class="busuanzi-count">
  <script async src="https://busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script>
    <span class="post-meta-item" id="busuanzi_container_site_uv" style="display: none;">
      <span class="post-meta-item-icon">
        <i class="fa fa-user"></i>
      </span>
      <span class="site-uv" title="Total Visitors">
        <span id="busuanzi_value_site_uv"></span>
      </span>
    </span>
    <span class="post-meta-divider">|</span>
    <span class="post-meta-item" id="busuanzi_container_site_pv" style="display: none;">
      <span class="post-meta-item-icon">
        <i class="fa fa-eye"></i>
      </span>
      <span class="site-pv" title="Total Views">
        <span id="busuanzi_value_site_pv"></span>
      </span>
    </span>
</div>








      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js"></script>
  <script src="/lib/velocity/velocity.min.js"></script>
  <script src="/lib/velocity/velocity.ui.min.js"></script>

<script src="/js/utils.js"></script>

<script src="/js/motion.js"></script>


<script src="/js/schemes/muse.js"></script>


<script src="/js/next-boot.js"></script>

<script src="/js/bookmark.js"></script>




  




  
<script src="/js/local-search.js"></script>













  

  
      

<script>
  if (typeof MathJax === 'undefined') {
    window.MathJax = {
      loader: {
        source: {
          '[tex]/amsCd': '[tex]/amscd',
          '[tex]/AMScd': '[tex]/amscd'
        }
      },
      tex: {
        inlineMath: {'[+]': [['$', '$']]},
        tags: 'ams'
      },
      options: {
        renderActions: {
          findScript: [10, doc => {
            document.querySelectorAll('script[type^="math/tex"]').forEach(node => {
              const display = !!node.type.match(/; *mode=display/);
              const math = new doc.options.MathItem(node.textContent, doc.inputJax[0], display);
              const text = document.createTextNode('');
              node.parentNode.replaceChild(text, node);
              math.start = {node: text, delim: '', n: 0};
              math.end = {node: text, delim: '', n: 0};
              doc.math.push(math);
            });
          }, '', false],
          insertedScript: [200, () => {
            document.querySelectorAll('mjx-container').forEach(node => {
              let target = node.parentNode;
              if (target.nodeName.toLowerCase() === 'li') {
                target.parentNode.classList.add('has-jax');
              }
            });
          }, '', false]
        }
      }
    };
    (function () {
      var script = document.createElement('script');
      script.src = '//cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js';
      script.defer = true;
      document.head.appendChild(script);
    })();
  } else {
    MathJax.startup.document.state(0);
    MathJax.texReset();
    MathJax.typeset();
  }
</script><!-- hexo-inject:begin --><!-- hexo-inject:end -->

    

  

</body>
</html>
